src/common/stats/StatsManager.cpp

```diff
@@ -249,6 +249,36 @@ void StatsManager::readAllValue(folly::dynamic& vals) {
                 vals.push_back(std::move(stat));
             }
         }
+
+        // add Percentile
+        for (auto range = TimeRange::FIVE_SECONDS; range <= TimeRange::ONE_HOUR;
```

Comment:
I want to know why not add one StatsMethod kind such as P99 directly?

Reply:
`p99` does not belong to the `StatsMethod`, it belong to HistogramMethod.

Reply:
make sense. Maybe we need HistogramMethod kind such as p50, p90, p95, p99

Reply:
Good point, 2.0 can add it.